### DISCUSSION TITLE
Wrong method being called on simple component

### DISCUSSION THREAD

**Discussion by TedPubC** [livewire/livewire#5960] [7. Bugs (archived)]
Testing out v3 and instantly found this bug - clicking "Counter 1" increments the first counter, clicking "Counter 2" also increments the first counter. 

<img width="1156" alt="image" src="https://github.com/livewire/livewire/assets/104569768/ad95c3ac-93d2-442e-9feb-3f8eba59e251">

This doesn't seem to the be the case if the counters are within unique elements (eg, adding any property to the HTML element, or prefixing the values with some text like `${{ $count1 }}` will fix the issue).

**Comment by nleibbrandt** [comment]
If you add unique wire:key to each button it will work, check https://laravel-livewire.com/docs/2.x/troubleshooting#dom-diffing-cures

**Comment by joshhanley** [comment]
@TedPubC yeah there is a bug in how morph works at the moment. See PR #5889 for more details. I think this issue is related.
